public/stylesheets/overlays/generic/generic.css

```diff
@@ -0,0 +1,647 @@
+    #main-container.empty-layout .lScore, 
+    #main-container.empty-layout .rScore, 
+    #main-container.empty-layout .rPlayer, 
+    #main-container.empty-layout .rPort, 
+    #main-container.empty-layout .rState, 
+    #main-container.empty-layout .rIcons  {
+        display: none;
+    }
+
+/* player-layout */
+
+    #main-container.player-layout .playerText {
+        position: absolute;
+        color: white;
+        font-size: 42px;
+        font-family: 'Oswald', cursive;
+        text-shadow: 2px 2px 2px rgb(50, 50, 50);
+
+        /* Oswald sits a little low, so raise the baseline */
+        margin-top: -5px;
+
+        overflow: hidden;
+        overflow-wrap: break-word;
+    }
+
+    #main-container.player-layout .title {
+        top: 0px;
+        width: 420px;
+        left: 750px;
+        height: 60px;
+        text-align: center;
+    }
+
+    #main-container.player-layout .tourneyInfo,
+    #main-container.player-layout .commentators,
+    #main-container.player-layout .lCharacter,
+    #main-container.player-layout .rCharacter,
+    #main-container.player-layout .lIcons,
+    #main-container.player-layout .lState, 
+    #main-container.player-layout .lPort,   
+    #main-container.player-layout .lPlayer, 
+    #main-container.player-layout .lScore, 
+    #main-container.player-layout .rScore, 
+    #main-container.player-layout .rPlayer, 
+    #main-container.player-layout .rPort, 
+    #main-container.player-layout .rState, 
+    #main-container.player-layout .rIcons  {
+        display: none;
+    }
+
+/* commentator-layout */
+
+    #main-container.commentator-layout .playerText {
+        position: absolute;
+        color: white;
+        font-size: 42px;
+        font-family: 'Oswald', cursive;
+        text-shadow: 2px 2px 2px rgb(50, 50, 50);
+
+        /* Oswald sits a little low, so raise the baseline */
+        margin-top: -5px;
+
+        overflow: hidden;
+        overflow-wrap: break-word;
+    }
+
+    #main-container.commentator-layout .title {
+        top: 0px;
+        width: 420px;
+        left: 750px;
+        height: 60px;
+        text-align: center;
+    }
+
+    #main-container.commentator-layout .commentators {
+        top: 814px;
+        left: 555px;
+        height: 60px;
+        width: 810px;
+        text-align: center;
+    }   
+
+    #main-container.commentator-layout .tourneyInfo,
+    #main-container.commentator-layout .lCharacter,
+    #main-container.commentator-layout .rCharacter,
+    #main-container.commentator-layout .lIcons,
+    #main-container.commentator-layout .lState, 
+    #main-container.commentator-layout .lPort,  
+    #main-container.commentator-layout .lPlayer, 
+    #main-container.commentator-layout .lScore, 
+    #main-container.commentator-layout .rScore, 
+    #main-container.commentator-layout .rPlayer, 
+    #main-container.commentator-layout .rPort, 
+    #main-container.commentator-layout .rState, 
+    #main-container.commentator-layout .rIcons  {
```

Comment:
This is starting to get redundant. I would just make a class on all of these elements.